src/containers/find-course/courses-filter-bar/CoursesFilterBar.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTranslation } from 'react-i18next'
+import { Box } from '@mui/material'
+
+import AppSelect from '~/components/app-select/AppSelect'
+import { sortTranslationKeys } from '~/containers/find-course/courses-filter-block/CoursesFilterBlock.constants'
+import { styles } from '~/containers/find-course/courses-filter-bar/CoursesFilterBar.styles'
+
+const CoursesFilterBar = () => {
+  const { t } = useTranslation()
+
+  const handleSortBy = (value: string) => {
+    console.log(value)
```

Comment:
do we need this `console.log()`?

Reply:
No, but for now we need this function handleSortBy, the functionality of this function will be done when we create at least one course.